moped-editor/src/layouts/DashboardLayout/TopBar.js

```diff
@@ -1,16 +1,13 @@
 import React, { useState } from "react";
 import { Link as RouterLink } from "react-router-dom";
-import clsx from "clsx";
-import PropTypes from "prop-types";
```

Comment:
I believe that, once we dispense of all uses of `makeStyles`, we'll be able to uninstall both of these libraries as well!